data/releases/eol.yaml

```diff
@@ -0,0 +1,66 @@
+branches:
+  - release: 1.21
+    finalPatchRelease: 1.21.14
+    endOfLifeDate: 2022-06-28
+  - release: "1.20"
```

Comment:
I'd quote all the minor and patch releases, so that parsers see these values as strings.